pcsx2/gui/AppCorePlugins.cpp

```diff
@@ -370,7 +370,7 @@ bool AppCorePlugins::OpenPlugin_GS()
 void AppCorePlugins::ClosePlugin_GS()
 {
 	_parent::ClosePlugin_GS();
-	if( CloseViewportWithPlugins && GetMTGS().IsSelf() && GSopen2 ) sApp.CloseGsPanel();
+	if( GetMTGS().IsSelf() && GSopen2 ) sApp.CloseGsPanel();
```

Comment:
I don't understand this change, it seems unrelated to the recording stuff completely.  Is this not changing existing GUI functionality in some way?